storage/aptosdb/src/transaction_store/mod.rs

```diff
@@ -78,16 +80,80 @@ impl TransactionStore {
         ))
     }
 
+    // TODO[Orderless]: Update this so that the user can specify even the range of chain timestamps
+    pub fn get_account_transaction_summaries_iter(
+        &self,
+        address: AccountAddress,
+        start_version: Option<u64>,
+        end_version: Option<u64>,
+        limit: u64,
+        ledger_version: Version,
+    ) -> Result<AccountTransactionSummariesIter> {
+        // Question[Orderless]: When start version is specified, we are current scanning forward from start version.
+        // When start version is not specified we are scanning backward, so as to return the most recent transactions.
+        // This doesn't seem to be a good design. Should we instead let the API take scan direction as input?
```

Comment:
how about always backward

(and only `end_version: Option<Version>, limit: u64`?)

Reply:
For transaction emitter, I'm actually using the `start_version`.